temba/msgs/models.py

```diff
@@ -1181,6 +1189,9 @@ def as_task_json(self):
 
         if self.contact_urn.auth:
             data.update(dict(auth=self.contact_urn.auth))
+            
+        if self.additional_params:
+            data.update(dict(additional_params=self.additional_params))
```

Comment:
Same here, simple setting of the parameter would be good. (same above with auth)